pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -122,7 +122,9 @@ public class KitRequestShipping extends KitRequest implements HasDdpInstanceId {
                     + "LEFT JOIN ddp_participant_exit ex ON (ex.ddp_instance_id = request.ddp_instance_id "
                     + "AND ex.ddp_participant_id = request.ddp_participant_id) "
                     + "LEFT JOIN ddp_kit_request_settings dkc ON (request.ddp_instance_id = dkc.ddp_instance_id "
-                    + "AND request.kit_type_id = dkc.kit_type_id) WHERE ex.ddp_participant_exit_id is null";
+                    + "AND request.kit_type_id = dkc.kit_type_id) "
+                    + "LEFT JOIN kit_type ktype ON (request.kit_type_id = ktype.kit_type_id) "
```

Comment:
Two things:

1. It is pretty much impossible to visually inspect a query this complex for a code review. It should really be refactored at some point.
2. I'm not sure what this new join provides since the test above is definitive.

We really need an automated test for this unless we have another way of evaluating that this is correct.

Reply:
Also confirmed with Kiara, we have general automated tests for kits :)